libraries/AP_Scripting/applets/set_home_position.lua

```diff
@@ -0,0 +1,41 @@
+-- Script to set new home position with the RC
+-- Sets the preset or current home position after 2 seconds that the switch is in the chosen position
+-- Author: Marco Robustini
+
+local rc_option = 300
+local channel = rc:find_channel_for_option(rc_option)
+  if not channel then
+return
+local sw_state = -1
+
+-- Define and initialize the location object with desired coordinates; enter the coordinates with the first 9 numbers and the height in centimetres
+my_home_location = Location()
+my_home_location:lat(448060084)
+my_home_location:lng(116100275)
+my_home_location:alt(0000)
```

Comment:
I think we would be better to leave `my_home_location` uninitialized and have the script deal with that. Then let the user fill it in if they want to. Otherwise users will not change it had then have a bad  time.
```suggestion
local my_home_location = Location()
--my_home_location:lat(448060084)
--my_home_location:lng(116100275)
--my_home_location:alt(0000)
```